### PR TITLE
Add more bonus tiers tests

### PR DESCRIPTION
**Pull request by al3xand3r-K**
New tests are based on boundary values analysis and equivalence partitioning.
- add more tests
- unify descriptions and titles of tests
- add `.idea` to `.gitignore`

### CODE REVIEW

File: test/BonusTiers.js [hunk 1]
```diff
@@ -17,78 +17,161 @@ contract('BonusTiers', function([owner, participant, wallet, proxy]) {
...
+  // bonuses expected:	25K
+  it('buy 2M on the edge of 3rd & 4th tiers', async function() {
+    const amount = web3.toWei(2000000, 'ether');
+    const sold = web3.toWei(74000000, 'ether');
+    const bonus = await shopTokenSale.getBonusTokens(amount, sold);
+    assert.equal(web3.fromWei(bonus, 'ether'), 25000, "Should get 25K bonus tokens");
+  });
+
+  // tokens sold:				24M
+  // tokens sought:			2M
```
Comment (pavel-kiselyov) [inline test/BonusTiers.js]: @al3xand3r-K only typo I've found: we're buying 27M here
